### DISCUSSION TITLE
Unable to install the Hellfire expansion

### DISCUSSION THREAD

**Discussion by DAllisterM** [diasurgical/DevilutionX#4062] [General]
I have been trying to transfer the files to the location but for some reason nothing will move it keep making the I can't do that noise and I have give it full promisons the game loads but I don't have the hellfire expansion it is just the base game available

**Comment by qndel** [comment]
You've literally provided 0 info about your platform and about what you are trying to do. Bonus points for broken english ...
What files, what location, what noise, what are "promisons" ...

**Reply by yuripourre** [reply]
promisons = permissions

**Reply by qndel** [reply]
> promisons = permissions

I don't speak goblin ;)

**Comment by AJenbo** [comment]
@DAllisterM in case you are not aware of it DevilutionX works on about 15 different systems, so we need you to let us know which one you are using.

**Comment by DAllisterM** [comment]
Android

**Reply by AJenbo** [reply]
Connect your device to a computer via USB, allow data transfer on the device. Then drag and drop `hellfire.mpq`, `hfmonk.mpq`, `hfmusic.mpq`, `hfvoice.mpq` from your Hellfire installation and `DIABDAT.MPQ` from Diablo to to `Android/data/org.diasurgical.devilutionx/files` on the Android device.

See here for the full installation instructions: https://github.com/diasurgical/devilutionX/blob/master/docs/installing.md#installing

If you still have issues then please post a picture of what files you have transfered and where you placed them. Also make sure you are running 1.3.0 and not an older beta.
